cloudinit/config/cc_ubuntu_pro.py

```diff
@@ -1,6 +1,6 @@
 # This file is part of cloud-init. See LICENSE file for license information.
 
-"""ubuntu_advantage: Configure Ubuntu Advantage support services"""
+"""ubuntu_pro: Configure Ubuntu Pro support services"""
```

Comment:
yes let's. I held off because of size of the diff, but renaming irrelevant ua_ names to pro_ now. Confirmed that latest pro client still honors ua_apt_http_proxy* config vars and stil writes `/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy`